src/machine.rs

```diff
@@ -241,13 +241,16 @@ pub struct Evaluator<'tcx> {
     /// The "time anchor" for this machine's monotone clock (for `Instant` simulation).
     pub(crate) time_anchor: Instant,
 
+    /// The virtual call stack.
+    pub(crate) stack: Vec<Frame<'mir, 'tcx, Tag, FrameData<'tcx>>>,
```

Comment:
I am not sure what is "virtual" about this -- it's certainly not more virtual than anything else Miri does. ;)